posts/_posts/2020-05-25-upgrade-guide.md

```diff
@@ -8,13 +8,11 @@ There are a number of breaking changes to be aware of when upgrading your Fabric
 
 **Upgrading from v3.x to v4.x** [(v4-breaking-changes)](v4-breaking-changes)
 
-Although the v4.x branch is at present still in beta, there are quite a few breaking changes that mostly involve the removal of old methods. If you make these changes now you can still run your project on the v3.x branch until v4.x leaves beta.
+Fabric.js v4.x left beta in August of 2020, and with it came quite a few breaking changes that mostly involve the removal of old methods. Rejoice because most of the breaking changes are small and of simple solution.
 
-Rejoice because most of the breaking changes are small and of simple solution.
+One big change is that the `clipTo` method which was depreciated in v2.4.0 was finally been removed, so if you're still using  `clipTo`  you will need to make the move to using  `clipPath`.
```

Comment:
```suggestion
One big change is that the `clipTo` method which was deprecated in v2.4.0 has finally been removed, so if you're still using  `clipTo`  you will need to make the move to using  `clipPath`.
```

Reply:
Just realized I've been misspelling deprecated for ages 😂